driver/main_args.ml

```diff
@@ -28,7 +29,11 @@ let mk_alert f =
     \    <alertname> can be 'all' to refer to all alert names";;
 
 let mk_absname f =
-  "-absname", Arg.Unit f, " Show absolute filenames in error messages"
+  "-absname", Arg.Unit f, " Show absolute filenames in error messages "
```

Comment:
I think it's a typo to have a space at the end of this string?
```suggestion
  "-absname", Arg.Unit f, " Show absolute filenames in error messages"
```

Reply:
Yes, It is indeed a typo, corrections will be made.